app/src/main/AndroidManifest.xml

```diff
@@ -2,6 +2,7 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.navapp">
 
+    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
```

Comment:
Why? It seems to be unused

Reply:
will need that to get current location l8r

Reply:
Again, if it's for later, add it later

Reply:
Left it by accident actually

Reply:
And you still left it